www/_posts/2017-04-21-cordova-7.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+author:
+    name: Audrey So
+    url: https://twitter.com/aud_rey_so
+title:  "Apache Cordova 7.0.0"
+categories: news
+tags: news releases
+---
+
+We are happy to announce that `Apache Cordova 7.0.0` has been released!
+
+Most notable changes include:
+* If a `package.json` does not exist in your project, it will be auto-created for you when `cordova prepare` is called.
+* When adding a platform or plugin, it will automatically save that platform or plugin to your `config.xml` and `package.json`. Details about platform and plugin versions are also automatically saved in `config.xml` and `package.json`. The `--save` flag is no longer required to save. Use `--nosave` to prevent saving to `config.xml` or `package.json`.
```

Comment:
Perhaps I would reword this to say that the previous default behaviour of not-saving has changed to saving? For me, when reading release notes, I want to understand how the new release impacts my use of the program in comparison to the prior release, so I appreciate calling out differences in default behaviour.